app/templates/resources/update_review.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td>{{ resource.website }}</td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.full_name() }}</td></tr>
+                <tr><td>Reviews</td><td>
+                    {% for review in resource.reviews.all() %}
+                        {{ review.content }}
+                    {% endfor %}
+                </td></tr>
+            </table>
+        </div>
+    </div>
+
+    <div class="ui grid container">
+        <div class="eight wide computer sixteen wide mobile centered column">
+            <h2 class="ui dividing header">Edit a review</h2>
+
+            {% set flashes = {
+                'error':   get_flashed_messages(category_filter=['form-error']),
+                'warning': get_flashed_messages(category_filter=['form-check-email']),
+                'info':    get_flashed_messages(category_filter=['form-info']),
+                'success': get_flashed_messages(category_filter=['form-success'])
+            } %}
+
+            {{ f.begin_form(form, flashes) }}
+
+                {{ f.render_form_field(form.rating) }}
+                {{ f.render_form_field(form.content) }}
+                {{ f.render_form_field(form.submit) }}
+
+                {{ f.form_message(flashes['error'], header='Something went wrong.', class='error') }}
+                {{ f.form_message(flashes['warning'], header='Check your email.', class='warning') }}
+                {{ f.form_message(flashes['info'], header='Information', class='info') }}
+                {{ f.form_message(flashes['success'], header='Success!', class='success') }}
+
+            {{ f.end_form(form) }}
+        </div>
+    </div>
+
+<script>
+    $( document ).ready(function() {
```

Comment:
I'm assuming you are doing this to put in the form the values that already exist in the database. That can be done in views.py like this https://github.com/hack4impact/reading-terminal-market/blob/master/app/vendor/views.py#L93